controllers/user/password.js

```diff
@@ -196,7 +196,7 @@ router
                         const username = req.user.userData.username;
                         const { password } = validationResult.value;
                         await Users.updateNewPassword({
-                            id: sanitise(username),
+                            id: req.user.userData.id,
```

Comment:
This fixes an important bug where for logged in users it wasn't actually updating the password.